### PR TITLE
chore: move e2e testsuite code to group together logically

### PR DESCRIPTION
**Pull request by damiannolan**
## Description

This PR just cut and pastes code. 

I found the `testsuite.go` file to be getting quite large. Moved some things around to group more logically together.

- Added `tx.go` for tx related testsuite functions
- Moved grpc query code to `grpc_query.go` and aligned import aliasing

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

**Comment by damiannolan** [comment]
It is really nice to see the CI all go green and have no unexpected or docker network failures ❤️

File: e2e/testsuite/grpc_query.go [hunk 1]
```diff
@@ -20,6 +28,63 @@ import (
...
+	ParamsQueryClient paramsproposaltypes.QueryClient
+	AuthQueryClient   authtypes.QueryClient
+	AuthZQueryClient  authz.QueryClient
+
+	ConsensusServiceClient tmservice.ServiceClient
+}
+
+// InitGRPCClients establishes GRPC clients with the given chain.
+// The created GRPCClients can be retrieved with GetChainGRCPClients.
+func (s *E2ETestSuite) InitGRPCClients(chain *cosmos.CosmosChain) {
```
Comment (DimitrisJim) [inline e2e/testsuite/grpc_query.go:55]: should [`GetChainGRPCClients`](https://github.com/cosmos/ibc-go/blob/47d678eb57d35a410f9a51baf47aa7a0d110df2c/e2e/testsuite/testsuite.go#L396) also be moved in this file?

Comment (chatton) [reply e2e/testsuite/grpc_query.go:55]: I think this makes sense, the one thing I see is that not all of the clients are query clients, there is one `ConsensusServiceClient` that is being initialised here too.

I wonder would it make sense to have two separate functions init-ing query/service clients and then calling then `InitGRPCClients` calling both.

**Review by chatton — APPROVED** [review]
LGTM

**Review by crodriguezvega — APPROVED** [review]
Nice re-organization, @damiannolan!

**Pull request merged**